src/js/view/controls/display-container.ts

```diff
@@ -4,12 +4,22 @@ import PlayDisplayIcon from 'view/controls/play-display-icon';
 import NextDisplayIcon from 'view/controls/next-display-icon';
 import { cloneIcons } from 'view/controls/icons';
 import { createElement } from 'utils/dom';
+import type Model from 'controller/model';
+import type { PlayerAPI } from 'types/generic.type';
+import type { Button } from './components/button';
+
+type Constructor<T extends {}> = new (...args: any[]) => T;
+type ButtonHolder = { [key: string]: Button | NextDisplayIcon };
 
 export default class DisplayContainer {
-    constructor(model, api) {
+    el: HTMLElement;
+    container: Element | null;
+    buttons: ButtonHolder;
+
+    constructor(model: Model, api: PlayerAPI) {
```

Comment:
The `_model` in view.js is an instance of `ViewModel`.

controller.js calls `_view = this._view = new View(_api, viewModel);`
view.js calls `controls.enable(_api, _model);` `_model` === `viewModel`
controls.js `const displayContainer = new DisplayContainer(model, api);` `model` === `viewModel`